torchmetrics/image/kid.py

```diff
@@ -112,6 +112,9 @@ class KernelInceptionDistance(Metric):
             Scale-length of polynomial kernel. If set to ``None`` will be automatically set to the feature size
         coef:
             Bias term in the polynomial kernel.
+        store_on_cpu: By default features will be calculated and stored on the device the metric is located on.
```

Comment:
quick ques: there are not just stored but also computed on CPU ?

if that's the case, do you think a better name might be `compute_on_cpu`?

Reply:
it is just stored, you always want to use GPU if possible, otherwise the metrics could be bottle-neck :)

Reply:
but I see, if they are stored on CPU, I think the computation will also happen CPU only.
if that's the case, then `store_on_cpu`, might not reflect the point that computation will also happen on CPU only.

or are they moved back to the device during computation?

Reply:
good point @SkafteNicki